styles/main.css

```diff
@@ -183,8 +183,16 @@ dt::after {
 .favourites [type="radio"] {
   display: none;
 }
-.favourites {
-  display: block;
+.favourites > label {
+  order: -1;
+  margin: 0 1em;
+  opacity: 50%;
+}
+.favourites > input:checked + label {
+  opacity: 100%;
+}
+.favourites > div {
+  display: none;
   width: 100%;
   color: blue;
```

Comment:
Some missing rules, and some improved styling to make it easier to distinguish which tab is currently selected